packages/app-runtime/test/runtime/AppStringProcessor.test.ts

```diff
@@ -1,23 +1,199 @@
-import { AppRuntime } from "../../src";
-import { TestUtil } from "../lib";
+import { ArbitraryRelationshipTemplateContentJSON } from "@nmshd/content";
+import { CoreDate } from "@nmshd/core-types";
+import { PeerRelationshipTemplateLoadedEvent } from "@nmshd/runtime";
+import assert from "assert";
+import { AppRuntime, LocalAccountSession } from "../../src";
+import { MockEventBus, MockUIBridge, TestUtil } from "../lib";
 
 describe("AppStringProcessor", function () {
-    let runtime: AppRuntime;
+    const mockUiBridge = new MockUIBridge();
+    const eventBus = new MockEventBus();
+
+    let runtime1: AppRuntime;
+    let runtime1Session: LocalAccountSession;
+
+    let runtime2: AppRuntime;
+    let runtime2SessionA: LocalAccountSession;
+
+    const templateContent: ArbitraryRelationshipTemplateContentJSON = { "@type": "ArbitraryRelationshipTemplateContent", value: "value" };
 
     beforeAll(async function () {
-        runtime = await TestUtil.createRuntime();
+        runtime1 = await TestUtil.createRuntime();
+        await runtime1.start();
+
+        const account = await TestUtil.provideAccounts(runtime1, 1);
+        runtime1Session = await runtime1.selectAccount(account[0].id);
+
+        runtime2 = await TestUtil.createRuntime(undefined, mockUiBridge, eventBus);
+        await runtime2.start();
+
+        const accounts = await TestUtil.provideAccounts(runtime2, 2);
+        runtime2SessionA = await runtime2.selectAccount(accounts[0].id);
+
+        // second account to make sure everything works with multiple accounts
```

Comment:
Do we really need this comment?

Reply:
Milena didn't got why there are two accounts so I added this comment.